open-machine-learning-jupyter-book/assignments/data-science/amazon-sagemaker-mlops-workshop-warm-up.md

```diff
@@ -0,0 +1,77 @@
+
+Then, enter [Amazon S3 URL](https://s3.amazonaws.com/aws-ai-ml-aod-latam/mlops-workshop/m.yml) to download the YAML file to the local.
+
+![SageMaker guide 2](../../../images/SageMaker-guide-2.png)
+
+After that, open the YAML file, since we do not need to use the high-end instance type like ml.m4.xlarge, we need to change InstanceType: "ml.m4.xlarge" to “ml.t2.medium” in the file.
+
+![SageMaker guide 3](../../../images/SageMaker-guide-3.png)
+
+```{note}
+If using ml.m4.xlarge, the notebook creation will not be successful.
+```
+
+Then, we can add the modified YAML file by clicking on the "Upload a template file" button.
+
+Subsequently, we need to set the NotebookInstanceSecGroupId parameter.
+
+![SageMaker guide 4](../../../images/SageMaker-guide-4.png)
+
+```{note}
+If a private subnet is not configured, SageMaker Notebook instances cannot access other resources within the same subnet or within a VPC with permissions.
+```
+
+In AWS CloudFormation, NotebookInstanceSecGroupId is a parameter used to specify the security group ID for SageMaker Notebook instances. Security groups are virtual firewalls used to control inbound and outbound network traffic to EC2 instances. Since SageMaker Notebook instances run on EC2 instances, a security group is required to control network traffic.
+The default value for the NotebookInstanceSecGroupId parameter is to select the "default security group". The default security group is automatically created when a VPC is created and allows all traffic to pass. If the default security group is selected, SageMaker Notebook instances can be accessed from all IP addresses.
+
+the NotebookInstanceSubnetId parameter is used to specify the subnet where SageMaker Notebook instances are deployed, improving security and availability by deploying them in the specified network location.
+
+Finally, wait for the stack to be created. The creation process may take a few minutes. When the stack status changes to "CREATE_COMPLETE", you can move on to the next step in the project.
+
+### Step 2 - open Jupyter Notebook
+
+Enter Amazon SageMaker, then enter Notebook instances, we can see that a new notebook is created.
+
+![SageMaker guide 5](../../../images/SageMaker-guide-5.png)
+
+Click "Open JupyterLab" to start the warmup exercises.
+
+### Step 3 - start warm-up section
+
+You can see the four files required to complete the warmup under the path lab/00_Warmup/.
+
+After running the code for uploading data, the data iris_train.csv and iris_test.csv will be uploaded to the Buckets of Amazon S3.
+
+![SageMaker guide 6](../../../images/SageMaker-guide-6.png)
+
+After running the code for training the model, the model will be trained in Amazon SageMaker. 
+Enter Training jobs to see the model (picture) being trained. 
+
+![SageMaker guide 7](../../../images/SageMaker-guide-7.png)
+
+The input data is the iris_train.csv previously created in Amazon S3.
+
+
+
+
+
+
+
```

Comment:
remove the unnecessary empty lines.

Reply:
Please add the acknowledgment section. And add the original license to the head of this file.

e.g. https://open-academy.github.io/machine-learning/prerequisites/python-programming-basics.html

Reply:
done